network-api/networkapi/wagtailpages/tests/libraries/rcc/test_forms.py

```diff
@@ -0,0 +1,302 @@
+import os
+
+from django.core import management
+from django.utils import translation
+
+from networkapi.wagtailpages.factory import profiles as profiles_factory
+from networkapi.wagtailpages.factory.libraries.rcc import (
+    detail_page as detail_page_factory,
+)
+from networkapi.wagtailpages.factory.libraries.rcc import relations as relations_factory
+from networkapi.wagtailpages.factory.libraries.rcc import (
+    taxonomies as taxonomies_factory,
+)
+from networkapi.wagtailpages.pagemodels.libraries.rcc import forms
+from networkapi.wagtailpages.pagemodels.libraries.rcc.forms import (
+    RCCLibraryPageFilterForm,
+)
+from networkapi.wagtailpages.tests.libraries.rcc import base as rcc_test_base
+from networkapi.wagtailpages.tests.libraries.rcc import utils as rcc_test_utils
+
+
+class TestFormUtilitiesFunctions(rcc_test_base.RCCTestCase):
+    def update_index(self):
+        with open(os.devnull, "w") as f:
+            management.call_command("update_index", verbosity=0, stdout=f)
```

Comment:
Doesn't look like this method is used anywhere in the class?

Reply:
Nice catch, thanks!